src/schemes/boundary/open_boundary/system.jl

```diff
@@ -482,6 +482,8 @@ function wrap_reference_function(constant_vector_, ::Val{NDIMS}) where {NDIMS}
     return constant_vector(coords, t) = SVector{NDIMS}(constant_vector_)
 end
 
+# The open boundary system either uses the fluids viscosity model or at the boundary the model
+# of the boundary which is the normal behavior to incorporate the boundaries impact in the viscosity term of the rhs.
```

Comment:
```suggestion
# The open boundary system either uses the fluid's viscosity model or, at the boundary, the model
# of the boundary, which is the normal behavior to incorporate the boundary's impact in the viscosity term of the rhs.
```
Can you please use the same comment in the other two locations as well?

Reply:
why the other two locations have nothing to do with the open boundary system?

Reply:
This comment has nothing to do with the open boundary system. It's the reason why we use the neighbor's viscosity and not the system's viscosity. But maybe we need to reformulate it to make this clearer?

Reply:
The first part of the sentence is only relevant for the open boundary system?

Reply:
Ah, oops. I didn't see that. Then my original suggestion?
```
# When available, use the viscosity of the neighbor system in the interaction.
# So, e.g. for fluid-boundary interaction, we use the boundary's viscosity, which can be used
# to model no-slip boundary conditions.
```
Plus the other suggestion below for the open boundary system.